problemsets-mat223/modules/module10-exercises.tex

```diff
@@ -1,10 +1,52 @@
 \begin{exercises}
+    %45 min
+    %60 min
 	\begin{problist}
-		\prob  Example problem with parts
+	    \prob Let $\mathcal R:\R^2\to\R^2$ be rotation clockwise by $135^\circ$, given by some matrix $M_R$. Compute $M_R$.
+	    
+	    \prob Let $\mathcal P:\R^2\to\R^2$ be projection onto the $y$-axis, given by some matrix $M_P$. Compute $M_P$.
+	    
+	    \prob Let $\mathcal F:\R^2\to\R^2$ be a transformation flipping over the line $y=x$, given by some matrix $M_F$. Compute $M_F$.
+	    
+	    \prob Let $\mathcal S:\R^2\to\R^2$ be a transformation stretching by 2, given by some matrix $M_F$. Compute $M_F$.
+	
+		\prob Let $\mathcal U:\R^2\to\R^2$ be the transformation given by $M=\matc{0&0\\-\sqrt{2}/2 & -\sqrt{2}/2}$, let $\mathcal R$ and $\mathcal P$ be the transformations given above.
 		\begin{enumerate}
-			\item   The first part
-			\item   The second part
+			\item   compute the matrix $M_1$ that gives $\mathcal R \circ \mathcal P$.
+			\item   compute the matrix $M_2$ that gives $\mathcal P \circ \mathcal R$.
+			\item   Write $\mathcal U$ as the composition (in some order) of $\mathcal R$ and $\mathcal P$.
 		\end{enumerate}
+		
+		\prob Let $\mathcal V:\R^2\to\R^2$ be the transformation given by $M=\matc{0&2\\2&0}$, let $\mathcal F$ and $\mathcal S$ be the transformations given above.
```

Comment:
Don't say "above". Use a `\label`/`\ref`to refer to the question or redefine the transformations